build/v1/types.go

```diff
@@ -742,6 +742,13 @@ type DockerBuildStrategy struct {
 	// policy. An additional experimental policy 'SkipLayersAndWarn' is the same as
 	// 'SkipLayers' but simply warns if compatibility cannot be preserved.
 	ImageOptimizationPolicy *ImageOptimizationPolicy `json:"imageOptimizationPolicy,omitempty" protobuf:"bytes,8,opt,name=imageOptimizationPolicy,casttype=ImageOptimizationPolicy"`
+
+	// volumes is a list of input volumes that can be mounted into buildah's runtime environment.
```

Comment:
don't mention buildah, it's implementation specific and "buildah's runtime environment" doesn't mean much to a user.  Explain it in terms of what it means for their consumption of the content.

Reply:
updated in next push